src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -1090,10 +1090,24 @@ private static void ReportImplicitImplementationMatchDiagnostics(Symbol interfac
                     else if (MemberSignatureComparer.RuntimeImplicitImplementationComparer.Equals(interfaceMember, member) && !member.IsAccessor())
                     {
                         // CONSIDER: Dev10 does not seem to report this for indexers or their accessors.
-                        diagnostics.Add(ErrorCode.WRN_MultipleRuntimeImplementationMatches, member.Locations[0], member, interfaceMember, implementingType);
+                        diagnostics.Add(ErrorCode.WRN_MultipleRuntimeImplementationMatches, GetDiagnosticLocation(member), member, interfaceMember, implementingType);
                     }
                 }
             }
+
+            Location GetDiagnosticLocation(Symbol member)
```

Comment:
>GetDiagnosticLocation [](start = 21, length = 21)

I think our naming convention for local function requires them to start with small letter, i.e. "**g**etDiagnosticLocation".